docs/documentation/upgrading/topics/keycloak/changes.adoc

```diff
@@ -1,5 +1,9 @@
 == Migration Changes
 
+=== Migrating to 25.0.0
+
+include::changes-25_0_0.adoc[leveloffset=3]
```

Comment:
Remind me, should we keep this commented out to prevent deployments to the site? Or does that only happen on release?

Reply:
This only happens at release time. We had some problem with the ad-hoc intermediate 21.1 release if I recall, but this shouldn't happen for any regular release.

Reply:
Thanks, good to know for future reference.